dotnet/src/IntegrationTests/Connectors/Memory/PgVector/PostgresVectorStoreRecordCollectionTests.cs

```diff
@@ -149,16 +149,15 @@ public async Task ItCanGetAndDeleteRecordAsync<TKey>(Type idType, TKey? key)
         var collectionName = "DeleteRecord";
         var sut = this.GetCollection(idType, collectionName);
 
-        await sut.CreateCollectionAsync();
+        await sut.EnsureCollectionExistsAsync();
```

Comment:
Huh, does this project not get build by default or something? I'm surprised the PR could be merged without any CI errors...